Makefile.am

```diff
@@ -1586,18 +1587,21 @@ __rpm_post:
 __rpm:
 	rpmbuild -ba $(RPM_SPEC_DIR)/cortx-motr.spec \
 		 --define "_topdir $(RPMBUILD_DIR)" \
+		 --define "CONFIG_X86_64 @ARCH@" \
```

Comment:
You might rename "CONFIG_X86_64" to "CONFIG_ARCH_TYPE".
Corresponding changes are needed for cortx-motr.spec.in

What do you think?
This can be done in next cycle.